MTGSDKSwift/MTGAPIManager.swift

```diff
@@ -137,33 +79,19 @@ final class NetworkOperation {
             }
             
             do {
-                let jsonResponse = try JSONSerialization.jsonObject(with: data!, options: []) as? JSONResults
-                
-                completion(jsonResponse, nil)
-                
+                let jsonResponse = try JSONSerialization.jsonObject(with: data!, options: [])
```

Comment:
let's do away with any force unwraps.  I know that logically, above you have the nil check, however if the code ever changes and that check gets moved or disappears, it has a high crash potential.  I'd prefer the following implementation:
```
guard let data = data else {
    return completion(Result.error(NetworkError.miscError("Network operation - No data returned")))
}

let jsonResponse = try JSONSerialization.jsonObject(with: data, options: [])
guard let json = jsonResponse as? JSONResults else {
        return completion(Result.error(NetworkError.miscError(NetworkError.miscError("Network operation - invalid json response")))
}

completion(Result.success(json))
```